sim/midas/src/main/scala/midas/platform/xilinx/AXI4Crossing.scala

```diff
@@ -170,6 +170,39 @@ class XilinxAXI4Bundle(val bundleParams: AXI4BundleParameters, val isAXI4Lite: B
       }
     }
   }
+
+  def tieoff(axiClock: Clock, axiReset: Bool): Unit = {
```

Comment:
```suggestion
  def tieoff(): Unit = {
```

So i know there are examples of this elsewhere, but this should be renamed to somthing like: `tieoffAsManager` or or something else to indicate this bidirectional is interface is being disabled from the manager-side if that makes sense.